src/writer/keyvalue_storage.py

```
@@ -0,0 +1,81 @@
+        return (agent_id, org_id)
+
+    def get(self, key: str, type_: Literal["data", "secret"]) -> Dict[str, Any]:
+        return self._request(partial(requests.get, url=f"{self.api_url}/agent_{type_}/{key}")).json()
+    
+    def save(self, key: str, data: Any) -> Dict[str, Any]:
+        try:
+            return self._create(key, data).json()
+        except requests.HTTPError as e:
+            if "already exists" in e.response.text:
+                return self._update(key, data).json()
+            raise e
```

Ruff (0.14.1)

Use `raise` without specifying exception name

Remove exception name

(TRY201)